### DISCUSSION TITLE
green videos

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#69] [General]
in previous and current  version, when playing mkv or other video files(mpv or vlc), usualy most combinations work, like 1080p x265 and x264, 2160p x264 and x265 but not if file is marked as hdr, it would show only green screen, its not as important but i thought  someone might want to know. 
also in 1.9 i have seen some youtube videos also just green screen, some facebook videos green screen, but generaly most of the videos work. cant say if its something i have done or something has been changed. sorry for not being too usefull with the details.

**Comment by Joshua-Riek** [comment]
The desktop environment uses wayland as it is the most optimized for the GPU. The downside is that HDR is not currently supported on wayland.

As for the green videos on Chromium I know precisely what you mean. Some videos are displayed as green when using the panfork mesa driver, but there is no current alternative for that driver. If this driver were removed, all 2D and 3D hardware acceleration would no longer work.

**Reply by unevoljitelj** [reply]
thanks for explanation, its cool, its not that bad.  but one annoying thing, since 1.9 usualy when using chromium or better to say when chromium is open i get hard locks, everything just freezes and only restart helps. again, not sure if this is on my side, but since i burned 1.9 on nvme i didnt experiment too much. only unusual thing i installed are apache 2 and mosquitto. and yeah its on nvme, so far i used it on sdcard.

**Reply by Joshua-Riek** [reply]
Weird, do you have at least a 5V/4A power supply? If the pi can not get enough power, things can freeze up.

**Reply by unevoljitelj** [reply]
at the moment its on 3amp charger. i used usb power meter to test on how much it draws and thought its enough, it usualy didnt pass 10w(2amp). it just occured to me when you mentioned psu that i didnt do it again with hardware acceleration fully available, maybe its drawing more power like this. ill go and try it out with more power. thanks again!

**Reply by Joshua-Riek** [reply]
Hopefully, that will fix the issue!
